pkg/hintrunner/zero/zerohint_signature.go

```diff
@@ -366,9 +367,33 @@ func newDivModNPackedDivmodV1Hint(a, b hinter.ResOperander) hinter.Hinter {
 				return err
 			}
 
-			valueBig := new(big.Int).Set(&resBig)
+			value_Big := new(big.Int).Set(&resBig)
+			res_Big := new(big.Int).Set(&resBig)
+			a_Big := new(big.Int).Set(&aPackedBig)
+			b_Big := new(big.Int).Set(&bPackedBig)
+			n_Big := new(big.Int).Set(&nBig)
+
+			if err := ctx.ScopeManager.AssignVariable("res", res_Big); err != nil {
```

Comment:
its all good except you dont need to assign `res` , just `value` is needed.

Reply:
But in the next hint, res is being used : 

`value = k = safe_div(res * b - a, N)`

Reply:
oh yeah you are right . my bad!